crypto/signature_nocgo.go

```diff
@@ -88,7 +88,10 @@ func Sign(hash []byte, prv *ecdsa.PrivateKey) ([]byte, error) {
 		return nil, errors.New("invalid private key")
 	}
 	defer priv.Zero()
-	sig := btc_ecdsa.SignCompact(&priv, hash, false) // ref uncompressed pubkey
+	sig, err := btc_ecdsa.SignCompact(&priv, hash, false) // ref uncompressed pubkey
```

Comment:
This change implies we somehow downgraded btcec to olde version with 2 return values?
Why do we have to do completely opposite change (changing from 2 to 1 return values) in ethermint and do the opposite in here? 
Not a big deal if it works, just curious.

Reply:
I don't see a call to `SignCompact` in Ethermint.

In Ethermint, we have to set the version of `btcec/v2` to `2.3.2` because otherwise we get this compilation error related to cometbft:
```
# github.com/cometbft/cometbft/crypto/secp256k1
../../../go/pkg/mod/github.com/!injective!labs/cometbft@v0.38.11-inj-0/crypto/secp256k1/secp256k1.go:131:14: assignment mismatch: 2 variables but ecdsa.SignComp
```
So the lowest common denominator version for btcec/v2 across our repos (go-ethereum, cosmos-sdk, etheremint, comet, core) is `v2.3.2`